src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+    <header>
+        <div class="leader">
+            <h1>Leaderboard</h1>
+        </div>
+    </header>
+    <main>
+        <section class="left">
+            <div class="recent-score">
+                <h2>Recent Scores</h2>
+                <button type="reset">Refresh</button>
+            </div>
+            <div class="scores">
+                <ul>
+                    <li>Lancelot : 100</li>
+                    <li>Lancelot : 100</li>
+                    <li>Lancelot : 100</li>
+                    <li>Lancelot : 100</li>
+                    <li>Lancelot : 100</li>
+                </ul>
+            </div>
+        </section>
+        <section class="right">
+            <div class="add-score">
+                <h2>Add your score</h2>
+                <form class="form" action="" method="post">
+                    <input type="text" placeholder="Your name"/>
+                    <input type="text" placeholder="Your score"/>
+                    <button type="submit">Submit</button>
+                </form>
+            </div>
+        </section>
+    </main>
```

Comment:
- [ ] Based on the wireframe shared with you these elements should cover enough space on the screen. In your design, most of the screen is left empty to the right. Kindly style your elements accordingly.

  ![image](https://user-images.githubusercontent.com/36542986/215774569-75a2f8b1-5019-47d3-9220-c5c9c2367396.png)